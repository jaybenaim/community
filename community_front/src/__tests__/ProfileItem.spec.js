import React from "react";
import { create } from "react-test-renderer";

describe("ProfileItem Component", () => {
  test("ProfileItem component has a label for the name", () => {
    const itemName = create(<label htmlFor="item name">Item Name</label>);
    expect(itemName.toJSON()).toMatchSnapshot();
  });
});
describe("ProfileItem Component", () => {
  test("ProfileItem component has a label for the price", () => {
    const itemPrice = create(<label htmlFor="item price">Item Price</label>);
    expect(itemPrice.toJSON()).toMatchSnapshot();
  });
});
describe("ProfileItem Component", () => {
  test("ProfileItem component has a label for the Address", () => {
    const itemAddress = create(
      <label htmlFor="item Address">Item Address</label>
    );
    expect(itemAddress.toJSON()).toMatchSnapshot();
  });
});
