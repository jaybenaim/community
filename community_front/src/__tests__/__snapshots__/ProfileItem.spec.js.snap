// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileItem Component ProfileItem component has a label for the Address 1`] = `
<label
  htmlFor="item Address"
>
  Item Address
</label>
`;

exports[`ProfileItem Component ProfileItem component has a label for the name 1`] = `
<label
  htmlFor="item name"
>
  Item Name
</label>
`;

exports[`ProfileItem Component ProfileItem component has a label for the price 1`] = `
<label
  htmlFor="item price"
>
  Item Price
</label>
`;
